addons/stock/__init__.py

```diff
@@ -19,6 +18,7 @@ def pre_init_hook(cr):
         ('module', '=', 'stock')
     ]).unlink()
 
+
```

Comment:
adding or removing newlines in a file without change is an unnecessary diff

to remove it you can checkout the file to previous version (`git checkout HEAD~1 -- addons/stock/__init__.py`) and amend then push